src/Chainweb/Mempool/Mempool.hs

```diff
@@ -746,7 +750,7 @@ getDecimal = do
 
 mockDecode :: ByteString -> Either String MockTx
 mockDecode s = do
-    s' <- B64.decode s
+    s' <- first T.unpack $ B64U.decodeBase64 $ B64.assertBase64 @'B64.UrlPadded @_ s
```

Comment:
I am a bit concerned about `B64.UrlPadded`. Does this mean the it uses padding? All use of base64 throughout chainweb should be base64Url without padding.